api/common.go

```diff
@@ -3,7 +3,7 @@ package api // import "github.com/docker/docker/api"
 // Common constants for daemon and client.
 const (
 	// DefaultVersion of the current REST API.
-	DefaultVersion = "1.49"
+	DefaultVersion = "1.50"
```

Comment:
We also need to update versions mentioned in the swagger file, e.g. https://github.com/moby/moby/blob/c04dec11437ff0c3f0238892c554c882dabb6235/api/swagger.yaml#L58-L60

"cleanest" would be to to both that, as well as the version-history.md in a single commit, like https://github.com/moby/moby/commit/4390ab275a19580f66cc4b068bd8a3b0b9a24356

Reply:
Done, thanks!